holmes/config.py

```diff
@@ -175,6 +182,24 @@ def _should_load_robusta_ai(self) -> bool:
 
         return True
 
+    def _auto_generate_transformer_configs(self) -> None:
+        """
+        Auto-generate transformer_configs from CLI fast_model and summarize_threshold parameters.
+        Only generates if fast_model is provided and transformer_configs is not already set.
+        """
+        if self.fast_model and not self.transformer_configs:
```

Comment:
Thinking here loud, if i understand correctly this means that when I use cli and pass the fast_model arg it will eventually add all toolsets trasformer using the `_apply_global_transformer_configs` is that corrent? 

If so im not sure that it is a good idea, instead can we only pass it to the transformers of each tool if fast_model wasn't defined on it?